pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>io.numaproj.numaflow</groupId>
     <artifactId>numaflow-java</artifactId>
-    <version>0.6.0</version>
+    <version>0.6.1</version>
```

Comment:
we are not yet ready to release a new version of JAVA SDK - let's sync up offline on this one.